runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/DataflowRunner.java

```diff
@@ -2303,7 +2303,8 @@ static String getDefaultContainerVersion(DataflowPipelineOptions options) {
   static boolean useUnifiedWorker(DataflowPipelineOptions options) {
     return hasExperiment(options, "beam_fn_api")
         || hasExperiment(options, "use_runner_v2")
-        || hasExperiment(options, "use_unified_worker");
+        || hasExperiment(options, "use_unified_worker")
+        || hasExperiment(options, "enable_prime");
```

Comment:
CC @robertwb spreading the knowledge that we are doing this. I think it is safe and adds value for users to be able to reject bundle finalizer usage on runner v1. Are there other pitfalls here? (other than any effect on future migrations)

Reply:
Yes, I think this is fine. Once the migration is complete we can remove these conditions.

Reply:
There seems to be another experiment not accounted for here: disable_prime_runner_v2

Reply:
I sent https://github.com/apache/beam/pull/16018 for this

Reply:
@tvalentyn, we should probably either revert this in the release branch, or cp #16018